docs/user-guide/kubernetes.md

```diff
@@ -112,66 +121,143 @@ spec:
-> With a multi node cluster we might expose Træfik with a NodePort or LoadBalancer service
-> and run more than 1 replica of Træfik for high availability.
+To deploy Træfik to your cluster start by submitting one of the YAML files to the cluster with `kubectl`:
 
-To deploy Træfik to your cluster start by submitting the deployment to the cluster with `kubectl`:
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-deployment.yaml
+```
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/containous/traefik/master/examples/k8s/traefik-ds.yaml
 ```
 
-### Check the deployment
+### Check the Pods
 
-Now lets check if our deployment was successful.
+Now lets check if our command was successful.
 
 Start by listing the pods in the `kube-system` namespace:
 
 ```shell
-$kubectl --namespace=kube-system get pods
+$ kubectl --namespace=kube-system get pods
 
 NAME                                         READY     STATUS    RESTARTS   AGE
 kube-addon-manager-minikubevm                1/1       Running   0          4h
 kubernetes-dashboard-s8krj                   1/1       Running   0          4h
 traefik-ingress-controller-678226159-eqseo   1/1       Running   0          7m
 ```
 
-You should see that after submitting the Deployment to Kubernetes it has launched
-a pod, and it is now running. _It might take a few moments for kubernetes to pull
+You should see that after submitting the Deployment or DaemonSet to Kubernetes it has launched
+a Pod, and it is now running. _It might take a few moments for kubernetes to pull
 the Træfik image and start the container._
 
 > You could also check the deployment with the Kubernetes dashboard, run
 > `minikube dashboard` to open it in your browser, then choose the `kube-system`
 > namespace from the menu at the top right of the screen.
 
-You should now be able to access Træfik on port 80 of your minikube instance.
+You should now be able to access Træfik on port 80 of your Minikube instance when using
+the DaemonSet:
 
 ```sh
 curl $(minikube ip)
 404 page not found
 ```
 
+If you decided to use the deployment, then you need to target the correct NodePort, which can
+be seen then you execute `kubectl get services --namespace=kube-system`.
+
+```sh
+curl $(minikube ip):<NODEPORT>
+404 page not found
+```
+
 > We expect to see a 404 response here as we haven't yet given Træfik any configuration.
 
 ## Deploy Træfik using Helm Chart
 
-Instead of installing Træfik via a Deployment object, you can also use the Træfik Helm chart.
+Instead of installing Træfik via an own object, you can also use the Træfik Helm chart. This
+allows more complex configuration via Kubernetes
+[ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configmap/) and enabled
+TLS certificates.
```

Comment:
Yes but I don't think that you are currently able to provide a TLS cert as a kubernetes secret..